man/systemd.device.xml

```diff
@@ -98,6 +98,12 @@
   </refsect1>
 
   <refsect1>
+    <title>Default Dependencies</title>
+
+    <para>There are no default dependencies for device units.</para>
```

Comment:
for the other pages you just added an xml comment. but for .device you added a section. Oversight or intended? (i think adding a comment instead is better)

Reply:
For unit type pages, I want to explicitly point out that there are no implicit/default dependencies because I mentioned them in systemd.unit(5).

For systemd.exec and systemd.resource-control, since they are only referenced in "Implicit Dependencies", I think it's okay to use comment.

So yes, it was intended. But I am open with changing them to comments.

Reply:
A few more words on my intention: when a user reads the sentence "For the implicit dependencies in each unit type, please refer to section "Implicit Dependencies" in respective man pages." in systemd.unit(5) and go see systemd.target(5) for the section "Implicit Dependencies", I want the user to know that target units don't have any implicit dependencies instead of being confused about "where is the Implicit Dependencies section?"